package.json

```diff
@@ -0,0 +1,30 @@
+{
+  "name": "leaderboard",
+  "version": "1.0.0",
+  "description": "",
+  "main": "index.js",
+  "scripts": {
+    "build": "webpack",
+    "dev": "webpack serve"
+  },
```

Comment:
- [ ] Great work on making the script but it would be nice if you add a key-value pair of `start` to your scripts object to make the user run your app easily by just running a familiar command `npm run start` or `npm start` 😊